.github/workflows/ci.yml

```diff
@@ -513,3 +514,22 @@ jobs:
         # TODO: this should become: `cargo hack wasi test --each-feature`
         run: cargo wasi test --test rt_yield --features wasi-rt
         working-directory: tests-integration
+
+  check-external-types:
+    name: check-external-types
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Rust ${{ env.rust_nightly }}
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: ${{ env.rust_nightly }}
+          override: true
+      - uses: Swatinem/rust-cache@v1
+      - name: check-external-types
+        run: |
+          set -x
+          cargo install cargo-check-external-types --locked --version 0.1.1
+          cargo check-external-types --config external-types.toml
```

Comment:
I think you need to use `--all-features` here. Otherwise, they seem to be checking only APIs enabled by default features, as far as I have tried in `slab` crate.

(Well, it is a little different from "only APIs enabled by default features" because of the cargo v1 resolver behavior, though.)

Reply:
Good catch! Definitely intended for that to be in there.